app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,30 @@
+class FriendshipsController < ApplicationController
+  def create
+    request = current_user.friendships.new(request_params)
+    request.status = true
```

Comment:
Why do you set the `request.status` to **true** when creating the friendship?
It's the confirmation that will set it to true.